Dockerfile

```diff
@@ -99,11 +98,7 @@ ENV DEBUG=${DEBUG:-""}
 # Install Python packages needed for development
 RUN if [ "$DEBUG" ]; then pip install -r requirements-dev.txt; fi;
 
-# Change user from root to invenio for better security
-RUN adduser --uid 1000 invenio --gid 0 && \
-    chown -R invenio:root /code
-USER 1000
```

Comment:
`inveniosoftware/centos7-python:3.6` uses `root` UID/GID by default. Removing this bit will keep using `root`. Are we OK with that?  (Well, deploying on OpenShift should be fine anyway I guess...)

Reply:
Good catch: This is actually good in the base image (see [here](https://github.com/inveniosoftware/docker-invenio/blob/b9f65fdae6758d2efdb1f303e1d94ecb8d65434e/python3.6/Dockerfile#L79) the problem is that we do not set the user properly in our Dockerfile. I am adding it, thanks!

Reply:
This hasn't been closed due to an incompatibility with `./run-tests.sh check-pytest`. To be addressed in a different PR.